File: day1.py
file = open("input.txt", "r")
lines = file.readlines()
number = 0
numbers = []
for line in lines:
    if line != "\n":
        number += int(line)
    else:
        numbers.append(number)
        number = 0

print(max(numbers))
numbers = sorted(numbers, reverse=True)
print(numbers)
print(numbers[0]+numbers[1]+numbers[2])

File: src/day1.rs
pub fn day1() {
    let input = std::fs::read_to_string("input1.txt").unwrap();
    let mut number = 0;
    let mut numbers = vec![];
    for line in input.lines() {
        if !line.is_empty() {
            number += line.trim().parse::<i32>().unwrap();
        } else {
            numbers.push(number);
            number = 0
        }
    }

    numbers.sort();
    numbers.reverse();
    println!("Solution Day1a: {}", numbers.first().unwrap());
    println!("Solution Day1b: {}", numbers[0] + numbers[1] + numbers[2])
}
